cmd/single-ocsp/main.go

```diff
@@ -110,6 +110,12 @@ func main() {
 	issuer, responder, target, pkcs11, err := readFiles(*issuerFile, *responderFile, *targetFile, *pkcs11File)
 	cmd.FailOnError(err, "Failed to read files")
 
+	if responder.NotAfter.Before(thisUpdate) {
```

Comment:
I'd like to reverse these so they read more easily:

`if thisUpdate.Before(responder.NotBefore)`

`if nextUpdate.After(responder.NotAfter)`

(also I think this fixed a logic bug in your code)